straxen/plugins/events/veto_proximity.py

```diff
@@ -99,6 +99,10 @@ def set_result_for_veto(self,
         res = self.get_overlapping_window_time(vetos_during_event, selected_intervals, event_window, result_buffer)
         result_buffer[f'veto_{veto_name}_overlap'] = res
         
+        # Check if the event windows overlap
+        _event_window_do_not_overlap = (strax.endtime(event_window)[:-1] - event_window['time'][1:]) <= 0
```

Comment:
What @reviewdog you are that smart?

Reply:
Don't really care, since this is exactly [the same implementation](https://github.com/AxFoundation/strax/pull/738/files) in `strax` which is the fastest we have.